### ISSUE TITLE
Screenshot icon - should it be something like one of these instead of a printer?

### ISSUE THREAD

**Issue by josephrbennett** [NCC-CNC/whattodo#37]
https://www.flaticon.com/free-icons/screenshot

**Comment by jeffreyhanson** [comment]
Yeah, I agree that the current icon isn't perfect and could potentially cause confusion. I had a look into possible ways to change the icon, and it seems that the R package that provides the screenshot button/functionality doesn't have the functionality to customize the icon for the screenshot button (see [here](https://trafficonese.github.io/leaflet.extras2/reference/easyprintOptions.html) for all the settings we can customize). 

So, in order to change the screenshot button icon, we would need to (1) re-implement the function entirely, (2) submit a PR to the package to add in the ability to customize the icon, or (3) fork the R package, tweak it, and then add it as a dependency. Although (3) is simplest option in the short term, this would increase the level of maintenance required for the app a lot (because the same R package also provides a lot of other functionality used by this app, so we would need to make sure any changes in the original version of the package also make it to the fork).

Given all that, I'm not sure if it's worth the hassle for changing the icon? What do you think @josephrbennett?

**Comment by jeffreyhanson** [comment]
I'll tag this as a sometime issue, but please feel free to update if you think this is important.

**Comment by jeffreyhanson** [comment]
Woops - sorry, I didn't mean to close the issue.

**Comment by josephrbennett** [comment]
Sounds fine to me. I didn't realize it'd be such a time-consuming thing. Thanks!
